inbound_test.go

```diff
@@ -143,6 +143,41 @@ func TestInboundConnection_CallOptions(t *testing.T) {
 	})
 }
 
+func TestCallOptionsPropogated(t *testing.T) {
```

Comment:
Let me know if this isn't the right spot for this integration test, wasn't sure of a better spot to put this.

Reply:
lgtm